docs/core-concepts/app-deployment.md

```diff
@@ -1,17 +1,255 @@
-# 🔧 .NET Core Application Deployment concepts
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can
-> track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/471) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this
-> topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on
-> [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
-	
\ No newline at end of file
+# .NET Core Application Deployment #
+
+You can deploy your .NET Core app in either of two ways:
+
+- As a portable app. A portable app relies on a shared system-wide version of the .NET Core framework that is present on a system.
+ 
+- As a self-contained application. All components, including .NET Core itself, are included with the application and are isolated from other .NET Core applications. 

```

Comment:
Should this article link to [the App Types article](https://dotnet.github.io/docs/core-concepts/app-types.html)?